0.data_analysis/construct_phenotypic_expression_plate_4_fs_data/correlation_relationships/README.md

```diff
@@ -1,11 +1,22 @@
-# Correlation Analysis
-In this analysis, the features values were compared for all cells.
+# Median Correlation Analysis
+In this analysis the aggreated features were compared across groups:
 ### Correlation Comparisons
-Pairwise correlations were calculated for the following groups:
-- Across Well within each construct concentration
-    - For the same siRNA constructs
-    - For different siRNA constructs
+Aggregated pairwise correlations were calculated across well between:
+- The same siRNA constructs within each construct concentration
+- Different siRNA constructs within each construct concentration
 
-- Within Well for each siRNA construct and construct concentration
+The `no_treatment` wells were only compared with other `no_treatment` wells for this analysis.
```

Comment:
Could you explain why? I think I am missing something here

Reply:
Thanks for catching that, this is something I will want to change in the other README as well. It is changed